tap_zendesk/streams.py

```diff
@@ -263,12 +266,18 @@ class Tickets(CursorBasedExportStream):
     replication_key = "generated_timestamp"
     item_key = "tickets"
     endpoint = "https://{}.zendesk.com/api/v2/incremental/tickets/cursor.json"
+    batch_size = DEFAULT_BATCH_SIZE
 
     def sync(self, state): #pylint: disable=too-many-statements
 
         bookmark = self.get_bookmark(state)
 
-        tickets = self.get_objects(bookmark)
+        # Fetch tickets with side loaded metrics
+        # https://developer.zendesk.com/documentation/ticketing/using-the-zendesk-api/side_loading/#supported-endpoints
+        tickets = self.get_objects(bookmark, side_load='metric_sets')
+
+        # Run this method to set batch size to fetch ticket audits and comments records in async way
+        self.check_access()
```

Comment:
Are we checking if metrics stream is selected before side-loading the stream?

Reply:
Yes, we are checking it [here](https://github.com/singer-io/tap-zendesk/pull/150/files#diff-89887ca52395ce152e9723c5595655edcd41793204060336e88ea9805c274433R310)